cnf-tests/.konflux/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG RHEL_VERSION=9.4
 
-FROM brew.registry.redhat.io/rh-osbs/openshift-ose-cli-rhel9:v4.19 AS oc
+FROM registry.redhat.io/openshift4/ose-cli-rhel9:v4.18 AS oc
```

Comment:
can we use the latest here ? (for 4.20 at this point)

Reply:
there is no 4.20 tag in production, latest one is 4.18. we need to update that once 4.19 is GA'd. This was discussed offline as well.

Reply:
we need anyhow to align with same OC version eventually so we need to update anyway. Not specifying the tag will give us the latest which is anyhow 4.18